external-crates/move/crates/move-binary-format/src/normalized.rs

```diff
@@ -793,8 +840,11 @@ impl<S: Hash + Eq> Struct<S> {
         self.type_parameters.iter().map(|param| &param.constraints)
     }
 
+    // Checks equivalence, omitting the defining module to avoid module name comparisons (which may
+    // be invalid during publication, etc).
     pub fn equivalent(&self, other: &Self) -> bool {
         let Self {
+            defining_module: _,
```

Comment:
Should we just check it? 
I had thought the conclusion to the meeting was that this can probably just be checked, and might be the better, future-proof option

Reply:
Check added in both places